back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -30,8 +30,16 @@ def index
       end
     user_baskets ||= {}
 
+    user_followers = current_user&.follows
+      &.where(followable_type: 'Project')
+      &.group_by do |follower|
+        [follower.followable_id, follower.followable_type]
+      end
+    user_followers ||= {}
```

Comment:
This piece of code occurs in many places, for example above you can see it for user baskets on top. Assuming there was a good reason for doing it like that and for consistency, I wrote this code the same way.

I think it would be nice if we could add some abstraction for passing things from controller to serializer. Picking the one relationship that belongs to the user (the user basket, the user vote, the user follow etc.) seems to be a common pattern.